ROADMAP.md

```diff
@@ -1,31 +1,37 @@
 # Roadmap
 
-## v0.5
+## v0.6
 
 ### Main Focus
 
-* [ ] (Waitset), event multiplexing and advanced push notifications
-* [ ] Finalize C and C++ API
-* [ ] Fully dynamic payload for pub/sub
-* [ ] ROS 2 rmw binding
-* [ ] Health monitoring
-* [ ] Reliability features for pub/sub
+The list is sorted by priority
+
+* [ ] Request/Response
+* [ ] Zenoh Gateway
+* [ ] Finalize C and C++ API (still some tiny things open)
+* [ ] Add `libc` based platform for better cross-compilation support
```

Comment:
I had a brief look at this over the weekend and I'm not sure anymore if this is a good way forward. I would remove it from the list for now.